src/client/pages/OfferNew/Checkout/InsuranceSummaryDetails.tsx

```diff
@@ -16,14 +16,15 @@ import { useCurrentLocale } from 'components/utils/CurrentLocale'
 import { Group, Row } from './InsuranceSummary'
 
 const Label = styled.div`
-  width: 50%;
   color: ${colorsV3.gray500};
 `
 const Value = styled.div`
-  width: 50%;
   color: ${colorsV3.gray700};
   text-align: right;
 `
+const HorizontalSpacer = styled.div`
+  width: 48px;
```

Comment:
Don't we use rems in this project?

Reply:
Since it doesn't really make sense the way we use `rem` I thought I could use `px` in this file where there are no other measurements. Also, we do have `px` in a bunch of other places. Isn't the important thing that we keep to the 4px scale?

Reply:
I still think we should continue to use `rem` for the new code we write 🙂

Reply:
I think we should "fasa ut det" since we use it weirdly 🙂